astropy/units/quantity.py

```diff
@@ -1784,8 +1784,28 @@ def choose(self, choices, out=None, mode="raise"):
         )
 
     # ensure we do not return indices as quantities
-    def argsort(self, axis=-1, kind="quicksort", order=None):
-        return self.view(np.ndarray).argsort(axis=axis, kind=kind, order=order)
+    def argsort(self, axis=-1, kind=None, order=None, *, stable=False):
```

Comment:
I guess an alternative would be to just pass `*args, **kwargs`? And then
Especially since `"quicksort"` is the default anyway.

Reply:
Oh I didn't realise that `kind=None` implied `"quicksort"`. In that case, might as well stick 100% to numpy's signature.